Xamarin.Forms.Platform.UAP/Xamarin.Forms.Platform.UAP.csproj

```diff
@@ -11,7 +11,7 @@
     <AssemblyName>Xamarin.Forms.Platform.UAP</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion>10.0.16299.0</TargetPlatformVersion>
+    <TargetPlatformVersion>10.0.18362.0</TargetPlatformVersion>
```

Comment:
Why is this happening?

Reply:
maybe because VS2019 installs this last/newest SDK by default and not all older Win10 SDKs.

Reply:
We had this before we rolled back to VS 2017 but the main reason has to do with WinUI functioning better when set to 10.0.18362.0 as the TargetPlatformVersion

And in theory it shouldn't matter increasing this since we still maintain the same mintarget

Reply:
@samhouts regardless though I just removed these changes. Because we no longer specify the build host on the YML we have to wait until all the build hosts are updated to VS2019 otherwise once we merge this everything will break on UWP